### DISCUSSION TITLE
Create ROI Polygon with  stream digitizing option

### DISCUSSION THREAD

**Discussion by Vyison** [semiautomaticgit/SemiAutomaticClassificationPlugin#376] [General]
Hello.

I am new and inexperienced, please forgive me if I am wrong here.
SCP is really powerful and extremely useful, even if I don't understand and use everything yet.
Is it possible to implement the option (as described in the title)? 
If this is already possible, please forgive me, I could not find/set the option.
I have attached a screenshot for your understanding (QGIS integrated digitizing mode).

Best regards
![streamDigi](https://github.com/user-attachments/assets/0e02c209-4c44-4a36-b6c8-c6b1c15d8e65)

**Comment by semiautomaticgit** [comment]
Hello @Vyison ,

thank you for your suggestion, I'll consider it in the future.
However, you can create polygons with QGIS and then import them in SCP by using this tool
https://semiautomaticclassificationmanual.readthedocs.io/en/latest/import_signatures_tab.html

**Reply by Vyison** [reply]
Hello Luca, 
thank you for your reply.
I'll follow your advice straight away and try it out.
Thank you very much
